src/uucore/src/lib/features/perms.rs

```diff
@@ -188,6 +189,77 @@ pub struct ChownExecutor {
     pub dereference: bool,
 }
 
+#[cfg(test)]
+pub fn check_root(path: &Path, would_recurse_symlink: bool) -> bool {
+    is_root(path, would_recurse_symlink)
+}
+
+fn is_root(path: &Path, would_recurse_symlink: bool) -> bool {
```

Comment:
Shouldn't it be `would_traverse_symlink` or is there some actual recursion going on here?

Reply:
You're right! I'll change the name of the variable.